local-storage/src/block.rs

```diff
@@ -0,0 +1,263 @@
+use anyhow::{bail, Result};
+use bytes::Bytes;
+use cid::Cid;
+use iroh_unixfs::Block;
```

Comment:
Do you know if they are keeping this repo around and/or if they have a new one?

Reply:
They are keeping the source around under a new [beetle](https://github.com/n0-computer/beetle) repo. I'm not sure how this will affect the crate though....I'll ask them about this.